config/examples/oauth2.yml

```diff
@@ -1,6 +1,6 @@
 base: &default
   github:
-    enabled: false
+    enabled: <%= ENV.fetch('GITHUB_ENABLED', true) %>
```

Comment:
wait, how is one supposed to disable this?

Reply:
Setting `GITHUB_ENABLED = 0` ? Not sure if that would work

Reply:
0 and "" are still truthy in ruby